README.md

```diff
@@ -15,9 +15,11 @@ You'll probably need `root` privileges to install the prerequisites.
    1. download and install [pip](https://pip.pypa.io/en/stable/installing/).
    2. install `virtualenv` using your package manager.
 
+3. By default, `virtualenv` uses the Python interpreter that it was installed with. If necessary, run `virtualenv --python=/usr/bin/python2.7 <path/to/new/virtualenv>` to have `virtualenv` use the 2.7 interpreter.
```

Comment:
"It was installed with" could possibly be confusing I think

Also "if necessary" but how do I know if it is necessary?